workspace/assets/css/lib/class-cursor.less

```diff
@@ -1,29 +1,25 @@
-.cursor-alias		{cursor: alias;}
-.cursor-all-scroll	{cursor: all-scroll;}
-.cursor-auto, 
-.cursor-normal		{cursor: auto;}
-.cursor-cell 		{cursor: cell;}
-.cursor-context-menu{cursor: context-menu;}
-.cursor-col-resize 	{cursor: col-resize;}
-.cursor-copy 		{cursor: copy;}
-.cursor-crosshair 	{cursor: crosshair;}
-.cursor-default 	{cursor: default;}
-.cursor-e-resize 	{cursor: e-resize;}
-.cursor-ew-resize,
-.cursor-ew 			{cursor: ew-resize;}
-.cursor-grab 		{cursor: grab;}
-.cursor-grabbing 	{cursor: grabbing;}
-.cursor-help 		{cursor: help;}
-.cursor-move, 
-.cursor-drag 		{cursor: move; cursor: -webkit-grab;}
-.cursor-no-drop 	{cursor: no-drop;}
-.cursor-none 		{cursor: none;}
-.cursor-not-allowed {cursor: not-allowed;}
-.cursor-pointer 	{cursor: pointer;}
-.cursor-progress 	{cursor: progress;}
-.cursor-row-resize 	{cursor: row-resize;}
-.cursor-text 		{cursor: text;}
-.cursor-wait,
-.cursor-loading 	{cursor: wait;}
-.cursor-zoom-in 	{cursor: zoom-in;}
-.cursor-zoom-out 	{cursor: zoom-out;}
\ No newline at end of file
+.cursor-alias                        {cursor: alias;}
+.cursor-all-scroll                   {cursor: all-scroll;}
+.cursor-auto,        .cursor-normal  {cursor: auto;}
+.cursor-cell                         {cursor: cell;}
+.cursor-context-menu                 {cursor: context-menu;}
+.cursor-col-resize                   {cursor: col-resize;}
+.cursor-copy                         {cursor: copy;}
+.cursor-crosshair                    {cursor: crosshair;}
+.cursor-default                      {cursor: default;}
+.cursor-e-resize                     {cursor: e-resize;}
+.cursor-ew-resize,   .cursor-ew      {cursor: ew-resize;}
+.cursor-grab                         {cursor: grab;}
+.cursor-grabbing                     {cursor: grabbing;}
+.cursor-help                         {cursor: help;}
+.cursor-move,        .cursor-drag    {cursor: move; cursor: -webkit-grab;}
+.cursor-no-drop                      {cursor: no-drop;}
+.cursor-none                         {cursor: none;}
+.cursor-not-allowed                  {cursor: not-allowed;}
+.cursor-pointer                      {cursor: pointer;}
+.cursor-progress                     {cursor: progress;}
+.cursor-row-resize                   {cursor: row-resize;}
+.cursor-text                         {cursor: text;}
+.cursor-wait,        .cursor-loading {cursor: wait;}
```

Comment:
Those should be split on separate lines.
Either:

```css
.cursor-wait,
.cursor-loading                                   {cursor: wait;}
```

or 

```css
.cursor-wait                                      {cursor: wait;}
.cursor-loading                                   {cursor: wait;}
```

I guess we really have a different view on what is readable ;)

Reply:
Dont want this. all other files are on one line by class declaration and if multiple class alias de base class they are all on the same line for better readability and validation of needed deprecated class